python/activator/visit.py

```diff
@@ -104,11 +107,11 @@ def get_bare_visit(self):
 @dataclass(frozen=True, kw_only=True)
 class SummitVisit(BareVisit):
     # Extra fields are in the NextVisit messages from the summit
-    private_efdStamp: float = 0.0
+    private_efdStamp: float = 0.0  # time of visit publication; UTC in unix seconds
     private_kafkaStamp: float = 0.0
     private_identity: str = "ScriptQueue"
     private_revCode: str = "c9aab3df"
     private_origin: int = 0
-    private_seqNum: int = 0
+    private_seqNum: int = 0        # queue-specific counter. Not the same as Butler/RubinTV seq_num
     private_rcvStamp: float = 0.0
-    private_sndStamp: float = 0.0
+    private_sndStamp: float = 0.0  # time of visit publication; TAI in unix seconds
```

Comment:
Trying to dig out again what these `private_*` fields mean and thinking whether adding more info would be useful. Just dumping them here

[Sasquatch](https://sasquatch.lsst.io/user-guide/timestamps.html)
> SAL timestamps are added automatically by SAL and have the private_ prefix. In particular, the private_sndStamp timestamp is the time at which the message was published by the CSC in TAI, which should be very close to the time at which the telemetry was measured. In fact, many telemetry topics rely on the private_sndStamp timestamp for that.
> The private_sndStamp timestamp is also converted to UTC and recorded in the private_efdStamp field, which is used as the time index in the EFD.


[slack thread](https://lsstc.slack.com/archives/CM65SGAJE/p1681916893293629)
> private_sndStamp, which should be the TAI instant when the SAL message was issued from the CSC, presumably only a short time after the actual event. 
> private_efdStamp should be the same value in UTC

And [slack thread](https://lsstc.slack.com/archives/CM65SGAJE/p1681918857875009?thread_ts=1681915811.873869&cid=CM65SGAJE)
> private_sndStamp ... is the time the sample was published.. this is TAI but is converted to UTC (and stored in private_efdStamp
> most topics will rely on private_sndStamp as also being the time the measurement was performed..

Reply:
After reading them again, I think what you've put in the comments are perfect summary for prompt processing context.

Reply:
Yes, I used the Sasquatch page as my source.

Though given the example of `private_seqNum`, we should probably avoid using the `private_` fields if possible. 😬